fallback-fb-setup/fallback-fb-setup.c

```diff
@@ -0,0 +1,56 @@
+#include <errno.h>
+#include <fcntl.h>
+#include <stdint.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <unistd.h>
+#include <sys/ioctl.h>
+#include <sys/mman.h>
+#include <sys/stat.h>
+#include <sys/types.h>
+#include <linux/fb.h>
+
+#define FBDEV "/dev/fb0"
+
+int main(int argc, char **argv) {
+    int ret = 0, fb_fd = -1;
+    uint8_t *p, *fbp = MAP_FAILED;
+    size_t screensize, bytes_per_pixel;
+    struct fb_var_screeninfo vinfo;
+
+    fb_fd = open(FBDEV, O_RDWR);
+    if (fb_fd == -1) {
+        perror("Failed to open " FBDEV);
+        ret = errno;
+        goto exit;
+    }
+
+    if (ioctl(fb_fd, FBIOGET_VSCREENINFO, &vinfo) == -1) {
+        perror("FBIOGET_VSCREENINFO failed on " FBDEV);
+        ret = errno;
+        goto exit;
+    }
+
+    bytes_per_pixel = vinfo.bits_per_pixel / 8;
+    screensize = vinfo.xres * vinfo.yres * bytes_per_pixel;
+
+    fbp = mmap(NULL, screensize, PROT_WRITE, MAP_SHARED, fb_fd, (off_t) 0);
+    if (fbp == MAP_FAILED) {
+        perror("Failed to mmap " FBDEV);
+        ret = errno;
+        goto exit;
+    }
+
+    for (p = fbp; p < fbp + screensize; p++)
```

Comment:
How about memset()?

Reply:
Yes, good point.

Reply:
I've now updated the PR to use `memset`.